testpilot/frontend/static-src/styles/modules/_experiment-list.scss

```diff
@@ -103,6 +103,10 @@
   }
 }
 
+.eol-message {
+  color: #f46323;
```

Comment:
We've been putting colors values in `testpilot/frontend/static-src/styles/_variables.scss`